cpp/src/io/json/write_json.cu

```diff
@@ -431,7 +433,9 @@ struct column_to_strings_fn {
         } else {
           return cudf::type_dispatcher(child_view.type(), *this, child_view);
         }
-      }();
+      };
```

Comment:
```suggestion
      }();
```

Reply:
It's written this way to avoid the temporary variable. It will help reduce peak memory.